added known issue for observability (#6503)
open-cluster-management/rhacm-docs: observability/customize_observability.adoc

For descriptions of all the parameters that can added into the `advanced` configuration, see the _Observability API_ documentation.

> [#keeping-retention-configuration]
> === Keeping retention configuration 

Reviewer: ```suggestion
```

> The default retention for all resolution levels, such as `retentionResolutionRaw,` retentionResolution5m`, or `retentionResolution1h`, is 365 days (365d). You must set an explicit value for the resolution retention in your `MultiClusterObservability` `spec.advanced.retentionConfig` parameter.

Reviewer: ```suggestion
The default retention for all resolution levels, such as `retentionResolutionRaw`, `retentionResolution5m`, or `retentionResolution1h`, is 365 days (`365d`). You must set an explicit value for the resolution retention in your `MultiClusterObservability` `spec.advanced.retentionConfig` parameter.
```

Reviewer: @douglascamata since the default is now `365d`, should the parameter values in the sample be changed to reflect that? @jc-berger I would move this info (lines 232-246) to line 224

> For {product-title-short} 2.11, the default retention for all resolution levels, such as `raw,` 5 minutes, or 1 hour, is updated to 365 days (365d). This 365d default retention means that the default retention for a 1 hour resolution has decreased from `indefinite` (`0d`) to `365d`. This retention change might cause you to lose data. If you did not set an explicit value for the resolution retention in your `MultiClusterObservability` `spec.advanced.retentionConfig` parameter, then this new default retention applies to you, and you might lose data.  

Reviewer: I recommend that you remove this subsection. Based on the explanation, it seems like the parameter values in the sample (line 207) should have the value, `365d`. Also that you move the following statement to line 196: "The default retention for all resolution levels, such as `retentionResolutionRaw,` retentionResolution5m`, or `retentionResolution1h` is 365 days (`365d`).  

At first I didn't understand where `raw`, 5 minutes, and 1 hour came from. For consistency, let's use the entire parameter name. We typically do not mention the version either.

```suggestion
The default retention for all resolution levels, such as `retentionResolutionRaw,` retentionResolution5m`, or `retentionResolution1h`, is 365 days (365d). You must set an explicit value for the resolution retention in your `MultiClusterObservability` `spec.advanced.retentionConfig` parameter.
```

If you upgraded from an earlier version and want to keep that version retention configuration, complete the following steps: 

> . Go to your application `MultiClusterObservability` resource. 

Reviewer: I think the user is able to simply search for the resource from the ACM console and then update it. @douglascamata please confirm

For descriptions of all the parameters that can added into the `advanced` configuration, see the _Observability API_ documentation.

The default retention for all resolution levels, such as `retentionResolutionRaw`, `retentionResolution5m`, or `retentionResolution1h`, is 365 days (`365d`). You must set an explicit value for the resolution retention in your `MultiClusterObservability` `spec.advanced.retentionConfig` parameter.

If you upgraded from an earlier version and want to keep that version retention configuration, complete the following steps: 

> . Go to your `MultiClusterObservability` resource by running the following `oc` binary in your CLI: 
> +
> [source,oc]
> ----
> edit mco observability
> ----
> . In the `spec.advanced.retentionConfig` parameter, apply the following configuration: 
> +
> [source,bash]
> ----
> spec:
>   advanced:
>     retentionConfig:
>       retentionResolutionRaw: 30d
>       retentionResolution5m: 180d
>       retentionResolution1h: 0d
> ----

Reviewer: ```suggestion
. Go to your `MultiClusterObservability` resource by running the following `oc` binary in your CLI: 

+
[source,bash]
----
edit mco observability
----

. In the `spec.advanced.retentionConfig` parameter, apply the following configuration: 

+
[source,bash]
----
spec:
  advanced:
    retentionConfig:
      retentionResolutionRaw: 30d
      retentionResolution5m: 180d
      retentionResolution1h: 0d
----
```
